docs/software/uenv/using.md

```diff
@@ -0,0 +1,445 @@
+By default, the modules are not activated when a uenv is started, and need to be explicitly activated using the `module` view.
+
+!!! example "using the module view"
+    ```console
+    $ uenv start prgenv-gnu/24.11:v1 --view=modules
+    $ module avail
+    ---------------------------- /user-environment/modules ----------------------------
+       aws-ofi-nccl/git.v1.9.2-aws_1.9.2    lua/5.4.6
+       boost/1.86.0                         lz4/1.10.0
+       cmake/3.30.5                         meson/1.5.1
+       cray-mpich/8.1.30                    nccl-tests/2.13.6
+       cuda/12.6.2                          nccl/2.22.3-1
+       fftw/3.3.10                          netlib-scalapack/2.2.0
+       fmt/11.0.2                           ninja/1.12.1
+       gcc/13.3.0                           openblas/0.3.28
+       gsl/2.8                              osu-micro-benchmarks/5.9
+       hdf5/1.14.5                          papi/7.1.0
+       kokkos-kernels/4.4.01                python/3.12.5
+       kokkos-tools/develop                 superlu/5.3.0
+       kokkos/4.4.01                        zlib-ng/2.2.1
+       libtree/3.1.1
+    $ module load cuda gcc cmake
+    $ nvcc --version
+    nvcc: NVIDIA (R) Cuda compiler driver
+    Cuda compilation tools, release 12.6, V12.6.77
+    $ gcc --version
+    gcc (Spack GCC) 13.3.0
+    $ cmake --version
+    cmake version 3.30.5
+    ```
+
+[](){#ref-uenv-error-v9modules}
+??? warning "bash: module: command not found"
+
+    Version 9.0.0 of uenv, installed on October 22 2025, has a bug that removes the module command on Santis and Clariden.
+
+    !!! note
+        The issue does not affect uenv in Slurm jobs, or on Daint and Eiger.
+
+    !!! note
+        A fix has been implemented, and will be installed as soon as possible.
+
+    Specifically, the `module` command is not available inside `uenv start` sessions:
+    ```console
+    $ uenv start prgenv-gnu/24.11:v2 --view=modules
+    $ module avail
+    bash: module: command not found
+    ```
+
+    The workaround is to manually load the module tool after starting your uenv session:
+
+    ```console
+    $ uenv start prgenv-gnu/24.11:v2 --view=modules
+    $ source /usr/share/lmod/8.7.17/init/bash
+    $ module avail
+
+    -------------- /user-environment/modules ---------------
+       aws-ofi-nccl/git.v1.9.2-aws_1.9.2
+       boost/1.86.0
+       cmake/3.30.5
+       cray-mpich/8.1.30
+       cuda/12.6.0
+       fftw/3.3.10
+       fmt/11.0.2
+       gcc/13.3.0
+       ...
+    ```
+
+
+[](){#ref-uenv-views-spack}
+### Spack view
+
+uenv images provide a full upstream Spack configuration to facilitate building your own software with Spack using the packages installed inside as dependencies.
+No view needs to be loaded to use Spack, however all uenv provide a `spack` view that sets some environment variables that contain useful information like the location of the Spack configuration, and the version of Spack that was used to build the uenv.
```

Comment:
side comment: we should enforce this somehow with stackinator or as a CI check in `alps-uenv`...